### ISSUE TITLE
Remove 'root' collection type and related code

### ISSUE THREAD

**Issue by damisul** [abartov/bybeconv#443]
We don't use 'root' collection type, but it is still declared and there are some leftovers of old code using it.
I propose to remove it to avoid confusion

**Comment by damisul** [comment]
@abartov , WDYT?

**Comment by abartov** [comment]
Yes. I was waiting to see if we can get away without the original design's "root collection" before removing that code. I think we can now do it.
